core/vibes/soul/sections/web-page-content/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { Stream, Streamable } from '@/vibes/soul/lib/streamable';
```

Comment:
I don't think this should be a vibes component or section, since webpages are only pertinent to Catalyst, but maybe someone disagrees!

Reply:
@jorgemoya I agree, I closed the vibes PR 👍